packages/flutter/test/painting/decoration_test.dart

```diff
@@ -279,7 +279,7 @@ void main() {
       '   direction provided in the ImageConfiguration object to match.\n'
       '   The DecorationImage was:\n'
       '     DecorationImage(SynchronousTestImageProvider(), center, match\n'
-      '     text direction)\n'
+      '     text direction, 1.0)\n'
```

Comment:
I think it's weird that it just says `1.0`. It should definitely say something like `scale: 1.0`.

Reply:
Makes sense, the above change you suggested will resolve this. I'll do the changes.